packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -414,106 +442,153 @@ export class Markets {
     );
 
     let filteredKeyedMarketCreatedLogs = keyedMarketCreatedLogs;
-    if (params.search) {
-      const fullTextSearchResults = await db.fullTextMarketSearch(params.search);
 
-      const keyedFullTextMarketIds: any = fullTextSearchResults.reduce(
-        (previousValue: any, fullTextSearchResult: any) => {
-          previousValue[fullTextSearchResult.market] = fullTextSearchResult;
-          return previousValue;
-        },
-        []
-      );
+    if (params.search || params.categories) {
+      let keyedFullTextResults: any = {};
+      let searchResults: any = [];
+      if (params.search) {
+        searchResults = await db.fullTextMarketSearch(params.search, null);
```

Comment:
Just a note we'll need to document wildcard character and any `advanced` search capabilities we get.

Reply:
Ok, I'm not sure if flex search supports the wildcard character or not. I'll have to check that.

Reply:
I'd like to know what is available in order to discuss with design. Maybe they want to offer an advanced search option. not pressing just curious.

Reply:
Yeah, sounds like we'll need to discuss this more at some point.